### ISSUE TITLE
evaluation에 대한 질문

### ISSUE THREAD

**Issue by Jaeminiman** [LISTatSNU/FastMRI_challenge#196]
안녕하세요.

실제 test data에 대한 평가는 제공된 reconstruct.py와 leaderboard_eval.py의 코드를 그대로 사용하여 진행하는 것으로 이해하면 될까요? (ex. SSIM loss를 구하는 방식)

감사합니다.

**Comment by wogur110** [comment]
네 맞습니다. 실제 test data는 leaderboard data와 동일한 포맷을 가지고 있으며, SSIM 값을 출력하는 방식은 leaderboard data와 동일하게 reconstruct.py 와 leaderboard_eval.py를 통해 출력합니다.

감사합니다.
배재혁 드림
